components/ModCreateFacilitySection.vue

```diff
@@ -75,13 +75,13 @@
                     />
                     <div class="flex flex-col mt-4">
                         <label
-                            for="Prefecture Japan"
+                            for="prefecture-select-en"
```

Comment:
I think you forgot to update the `for` attributes to match the element ids that the labels are bound to. [Documentation](https://www.w3schools.com/tags/att_label_for.asp)